### ISSUE TITLE
Interested in transfering @typst npm organization to a @typst member

### ISSUE THREAD

**Issue by jcbhmr** [typst/typst#2946]
Hello! 👋

I recently opened a PR to add an npm package as an install/distribution method for the Typst CLI binary. #2933 Unfortunately it wasn't viable due to scope creep. However, I would still like to transfer ownership of the https://www.npmjs.com/org/typst npm organization to the official @typst GitHub project at some point.

**Why?**
1. Establish same-name-everywhere brand identity. This is great for when Googling so that example.com/typst or example.com/user/typst or whatever is _the Typst project_ and there's no "is this official or not?" questions. Even just claiming the name and sitting on it is better than someone else hijacking it lol. Look what happened to https://www.npmjs.com/package/deno https://github.com/YounGoat/nodejs.deno/issues/5 (official project is https://www.npmjs.com/package/deno-bin)
2. Stop someone else from getting it. see above 👆
3. remove the "well we would need to do some admin work to acquire and create the @typst npm organization" hurdle from releasing Node.js/JavaScript language bindings for Typst or any other JavaScript & Typst -related projects. Just a thought.

in fact, this already is being done with the Cargo Rust typst package: https://crates.io/crates/typst

**TL;DR:** I currently have the https://www.npmjs.com/org/typst and I would like to transfer it to someone or some peoples at @typst.

If at some point the @typst project would like to have the https://www.npmjs.com/package/typst package name, I'd be happy to transfer it to you (it is the "typst" name after all lol), but it was indicated that such a npm Typst distribution or JavaScript API is off the _official_ table at present.

**Comment by laurmaedje** [comment]
That makes sense. Since the existing typst package also isn't part of it, it also doesn't add any maintainance overhead for us. I'm not sure how ownership transfer works on npm. Do you need an email address that is linked to an npm account?

**Comment by jcbhmr** [comment]
> I'm not sure how ownership transfer works on npm. Do you need an email address that is linked to an npm account?

Yep! I get this form:
![image](https://github.com/typst/typst/assets/61068799/c85d7441-07ae-4cd7-9ee6-18b3426d0573)

I need an npm username or the email that you used to sign up for npm of someone at @typst to add as a member+owner of the https://www.npmjs.com/org/typst npm organization.

**Comment by laurmaedje** [comment]
I made an account. It's the same username and email I use on GitHub.

**Comment by jcbhmr** [comment]
I sent an invite and it looks like you accepted! 🥳

<details><summary>I changed you from the default Member to Owner</summary>

![image](https://github.com/typst/typst/assets/61068799/6702f874-dc4f-45af-b400-e17aa48b5991)

</details>

Closing this as done! 🎉
@laurmaedje can now add/remove/delete/edit/etc. to add any other @typst members if/when needed.
